<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>view_datawindow_edit</class>
 <widget class="QWidget" name="view_datawindow_edit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>view_datawindow_edit</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_12">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_11">
     <item row="0" column="0">
      <widget class="QWidget" name="widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_add">
          <property name="text">
           <string>增加(&amp;A)</string>
          </property>
          <property name="icon">
           <iconset resource="ssp_gui.qrc">
            <normaloff>:/ssp_gui/Resources/20070402165925169.gif</normaloff>:/ssp_gui/Resources/20070402165925169.gif</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_copy">
          <property name="text">
           <string>复制(&amp;C)</string>
          </property>
          <property name="icon">
           <iconset resource="ssp_gui.qrc">
            <normaloff>:/ssp_gui/Resources/20070402165933688.gif</normaloff>:/ssp_gui/Resources/20070402165933688.gif</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_del">
          <property name="text">
           <string>删除(&amp;D)</string>
          </property>
          <property name="icon">
           <iconset resource="ssp_gui.qrc">
            <normaloff>:/ssp_gui/Resources/del.gif</normaloff>:/ssp_gui/Resources/del.gif</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_view">
          <property name="text">
           <string>预览(&amp;V)</string>
          </property>
          <property name="icon">
           <iconset resource="ssp_gui.qrc">
            <normaloff>:/ssp_gui/Resources/preview.gif</normaloff>:/ssp_gui/Resources/preview.gif</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_import">
          <property name="text">
           <string>导入(I)</string>
          </property>
          <property name="icon">
           <iconset resource="ssp_gui.qrc">
            <normaloff>:/ssp_gui/Resources/export.gif</normaloff>:/ssp_gui/Resources/export.gif</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_export">
          <property name="text">
           <string>导出(O)</string>
          </property>
          <property name="icon">
           <iconset resource="ssp_gui.qrc">
            <normaloff>:/ssp_gui/Resources/export2.gif</normaloff>:/ssp_gui/Resources/export2.gif</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_apply">
          <property name="text">
           <string>应用(&amp;L)</string>
          </property>
          <property name="icon">
           <iconset resource="ssp_gui.qrc">
            <normaloff>:/ssp_gui/Resources/hujiaozhuanyi.gif</normaloff>:/ssp_gui/Resources/hujiaozhuanyi.gif</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>165</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tab_1">
        <attribute name="title">
         <string>引用</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_5">
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item row="0" column="0">
          <widget class="QSplitter" name="splitter_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>主表</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <property name="leftMargin">
              <number>4</number>
             </property>
             <property name="topMargin">
              <number>4</number>
             </property>
             <property name="rightMargin">
              <number>4</number>
             </property>
             <property name="bottomMargin">
              <number>4</number>
             </property>
             <item row="0" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label_ref_cls">
                 <property name="text">
                  <string>引用分类</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboBox_ref_cls"/>
               </item>
               <item>
                <spacer name="horizontalSpacer_5">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <widget class="QTableWidget" name="tableWidget_ref1"/>
             </item>
            </layout>
           </widget>
           <widget class="QSplitter" name="splitter">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>SQL语句</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_3">
              <property name="leftMargin">
               <number>4</number>
              </property>
              <property name="topMargin">
               <number>4</number>
              </property>
              <property name="rightMargin">
               <number>4</number>
              </property>
              <property name="bottomMargin">
               <number>4</number>
              </property>
              <item row="1" column="0">
               <widget class="QTextEdit" name="textEdit_ref"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="layoutWidget">
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QWidget" name="widget_2" native="true">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>30</height>
                 </size>
                </property>
                <layout class="QGridLayout" name="gridLayout_2">
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <property name="spacing">
                  <number>6</number>
                 </property>
                 <item row="0" column="0">
                  <widget class="QPushButton" name="pushButton_ref2_add">
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>24</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>增加(&amp;N)</string>
                   </property>
                   <property name="icon">
                    <iconset resource="ssp_gui.qrc">
                     <normaloff>:/ssp_gui/Resources/20070402165925169.gif</normaloff>:/ssp_gui/Resources/20070402165925169.gif</iconset>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QPushButton" name="pushButton_ref2_del">
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>24</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>删除(&amp;L)</string>
                   </property>
                   <property name="icon">
                    <iconset resource="ssp_gui.qrc">
                     <normaloff>:/ssp_gui/Resources/del.gif</normaloff>:/ssp_gui/Resources/del.gif</iconset>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="2">
                  <spacer name="horizontalSpacer_2">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>76</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_3">
                <property name="title">
                 <string>对应行</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_4">
                 <property name="leftMargin">
                  <number>4</number>
                 </property>
                 <property name="topMargin">
                  <number>4</number>
                 </property>
                 <property name="rightMargin">
                  <number>4</number>
                 </property>
                 <property name="bottomMargin">
                  <number>4</number>
                 </property>
                 <item row="0" column="0">
                  <widget class="QTableWidget" name="tableWidget_ref2"/>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>数据集</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_13">
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item row="0" column="0">
          <widget class="QSplitter" name="splitter_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>主表</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_7">
             <property name="leftMargin">
              <number>4</number>
             </property>
             <property name="topMargin">
              <number>4</number>
             </property>
             <property name="rightMargin">
              <number>4</number>
             </property>
             <property name="bottomMargin">
              <number>4</number>
             </property>
             <item row="0" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="label_data_cls">
                 <property name="text">
                  <string>数据集分类</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboBox_data_cls"/>
               </item>
               <item>
                <widget class="QCheckBox" name="checkBox_useCls">
                 <property name="text">
                  <string>分类过滤引用选择</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_6">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <widget class="QTableWidget" name="tableWidget_data1"/>
             </item>
            </layout>
           </widget>
           <widget class="QSplitter" name="splitter_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <widget class="QGroupBox" name="groupBox_5">
             <property name="title">
              <string>SQL语句</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_6">
              <property name="leftMargin">
               <number>4</number>
              </property>
              <property name="topMargin">
               <number>4</number>
              </property>
              <property name="rightMargin">
               <number>4</number>
              </property>
              <property name="bottomMargin">
               <number>4</number>
              </property>
              <item row="0" column="0">
               <widget class="QTextEdit" name="textEdit_data"/>
              </item>
             </layout>
            </widget>
            <widget class="QTabWidget" name="tabWidget_2">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="tab_4">
              <attribute name="title">
               <string>列定义</string>
              </attribute>
              <layout class="QGridLayout" name="gridLayout_9">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item row="0" column="0">
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <property name="leftMargin">
                  <number>4</number>
                 </property>
                 <property name="topMargin">
                  <number>4</number>
                 </property>
                 <property name="rightMargin">
                  <number>4</number>
                 </property>
                 <property name="bottomMargin">
                  <number>4</number>
                 </property>
                 <item>
                  <widget class="QWidget" name="widget_3" native="true">
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>30</height>
                    </size>
                   </property>
                   <layout class="QGridLayout" name="gridLayout_10">
                    <property name="leftMargin">
                     <number>0</number>
                    </property>
                    <property name="topMargin">
                     <number>0</number>
                    </property>
                    <property name="rightMargin">
                     <number>0</number>
                    </property>
                    <property name="bottomMargin">
                     <number>0</number>
                    </property>
                    <item row="0" column="6">
                     <spacer name="horizontalSpacer_3">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>132</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item row="0" column="3">
                     <widget class="QPushButton" name="pushButton_newcol">
                      <property name="maximumSize">
                       <size>
                        <width>16777215</width>
                        <height>24</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>初始列生成</string>
                      </property>
                      <property name="icon">
                       <iconset resource="ssp_gui.qrc">
                        <normaloff>:/ssp_gui/Resources/20070402165925152.gif</normaloff>:/ssp_gui/Resources/20070402165925152.gif</iconset>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="4">
                     <widget class="QPushButton" name="pushButton_data2_add">
                      <property name="maximumSize">
                       <size>
                        <width>16777215</width>
                        <height>24</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>增加(&amp;N)</string>
                      </property>
                      <property name="icon">
                       <iconset resource="ssp_gui.qrc">
                        <normaloff>:/ssp_gui/Resources/20070402165925169.gif</normaloff>:/ssp_gui/Resources/20070402165925169.gif</iconset>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="5">
                     <widget class="QPushButton" name="pushButton_data2_del">
                      <property name="maximumSize">
                       <size>
                        <width>16777215</width>
                        <height>24</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>删除(&amp;L)</string>
                      </property>
                      <property name="icon">
                       <iconset resource="ssp_gui.qrc">
                        <normaloff>:/ssp_gui/Resources/del.gif</normaloff>:/ssp_gui/Resources/del.gif</iconset>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QGroupBox" name="groupBox_6">
                   <property name="title">
                    <string/>
                   </property>
                   <layout class="QGridLayout" name="gridLayout_8">
                    <property name="leftMargin">
                     <number>4</number>
                    </property>
                    <property name="topMargin">
                     <number>4</number>
                    </property>
                    <property name="rightMargin">
                     <number>4</number>
                    </property>
                    <property name="bottomMargin">
                     <number>4</number>
                    </property>
                    <item row="0" column="0">
                     <widget class="QTableWidget" name="tableWidget_data2"/>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tab_5">
              <attribute name="title">
               <string>条件定义</string>
              </attribute>
             </widget>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ssp_gui.qrc"/>
 </resources>
 <connections/>
</ui>
